nautobot/ui/src/utils/navigation.js

```diff
@@ -0,0 +1,33 @@
+import { Suspense } from "react";
+import { lazy } from "react";
+import { my_import_as_function } from "./app-import";
+import { LoadingWidget } from "@components/LoadingWidget";
+
+export function getPluginRoutes() {
+    let react_routes = [];
+    let app_routes = [];
+    try {
+        app_routes = require("../app_routes.json");
+    } catch (error) {
+        console.warn("app_routes.json not found");
+    }
```

Comment:
use of a try-and-catch method due to test-ui failures

Reply:
We should probably have an empty file or address this directly. The example plugin is loaded for tests so we shouldn't ignore this workflow.